docs/security-compliance/token-formats.mdx

```diff
@@ -25,13 +25,19 @@ security breaches and unauthorized access.
 
 Ory also issues session cookies to maintain user sessions across requests. Session cookies are usually used to store user
 authentication information, such as the user ID, and can be used to provide a seamless user experience. Ory session cookies are
-prefixed with the ory*session* prefix, making them easily identifiable in logs and tracking tools.
+prefixed with the `ory_session_` prefix, making them easily identifiable in logs and tracking tools.
 
 Session cookies are essential for maintaining user sessions, and Ory ensures that session cookies are secure and tamper-proof.
 Developers can configure the expiration time and cookie options for Ory session cookies to fit their application's specific needs.
 
-In summary, Ory session cookies are prefixed with ory*session* and are essential for maintaining user sessions. They can be
-customized to fit specific application needs and are secure and tamper-proof.
+## Other Ory cookies
+
+Ory Network will also issue other cookies to protect against CSRF attacks, provide load balancing, and other security features.
+These cookies will be prefixed as follows:
+
+- `ory_`: Ory-internal cookie (CSRF protection etc)
```

Comment:
```suggestion
- `ory_`: Ory-internal cookie (CSRF protection)
```

IMO either list explicitly or not (I assume its not that much)

Reply:
We have a bunch of other cookies which I'd only list of someone really, really, really needs to know their names for some inexplicable reason. GDPR certainly doesn't require listing them all by name.